app/uk/gov/hmrc/digitalservicestax/views/uniform/radios.scala.html

```diff
@@ -17,9 +17,17 @@
 @(key: List[String], options: Seq[String], existing: Option[String], errors: ErrorTree, messages: UniformMessages[Html], conditional: PartialFunction[String,Html] = PartialFunction.empty )
 
 @keyNoDots=@{key.mkString("-")}
-@inlineRadioKeys=@{(key.mkString(".") == "loan-amount.estimate")}
-
+@inlineRadioKeys=@{key.mkString(".") == "manage-companies"}
 @standard_field(key, errors, "radios", messages) {
+@if(inlineRadioKeys) {
+    <fieldset class="govuk-fieldset" id="add-company-to-group" aria-describedby="add-to-list-hint">
```

Comment:
I think we can avoid this by mapping over the messages. That way you could have optional content that appears/disappears based on the messages rather than needing to hard-code things into the views for a given field type.

Reply:
Will investigate this